man/gemfile.5.ronn

```diff
@@ -439,6 +459,12 @@ software is installed or some other conditions are met.
 
 ## GEMSPEC
 
+The [.gemspec](http://guides.rubygems.org/specification-reference/) file is where
+ you provide metadata about your gem to Rubygems, the Ruby ecosystem's gem
```

Comment:
`the Ruby ecosystem's hosting service` This sounds like you're talking about RubyGems.org which is separate from RubyGems